apps/cloud/src/api/user.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by acunningham on 22/05/17.
+ */
+import Store from '@raincatcher/store';
+import webapi, {ApiService, StoreApiService, WebApiConfig}  from '@raincatcher/webapi';
+import * as express from 'express';
+
+
+// Define User datatype
+interface User {
+  id: string,
+  username: string,
+  name: string,
+  position: string,
+  phone: string,
+  email: string,
+  notes: string,
+  avatar: string,
+  banner: string,
```

Comment:
our store draft doesn't support single nested documents, of course, so maybe that's a feature to look into